### DISCUSSION TITLE
Dark SDK

### DISCUSSION THREAD

**Discussion by medz** [minio/minio#14106] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Related, new language support

**Describe the solution you'd like**
Dart/Flutter is a hot technology at the moment. Many developers around the world use Flutter to create apps, and accessing cloud storage is a very troublesome thing. Except for Google Storage, which has an official Package, no other cloud storage has it. MinIO is a great choice!

**Comment by harshavardhana** [comment, marked as answer]
> Dart/Flutter is a hot technology at the moment. Many developers around the world use Flutter to create apps, and accessing cloud storage is a very troublesome thing. Except for Google Storage, which has an official Package, no other cloud storage has it. MinIO is a great choice!

Unfortunately we do not have the bandwidth right now to support another SDK, perhaps may be in future. Thanks for using MinIO.
